README.md

```diff
@@ -27,6 +27,46 @@ script/bootstrap
 
 ## Running the application
 
+To run the application there are two options.
+
+For either option the frontend can further decide if it's requests should be
+sent to the locally running API or to a wiremock version. To do this we change
+the environment variable `APPROVED_PREMISES_API_URL` within .env to either of
+the following values:
+
+* Fake API (via wiremock) - <http://localhost:9092>
+* [Real API](https://github.com/ministryofjustice/hmpps-approved-premises-api) -
+  <http://localhost:8080>
+
+### 1. Using AP Tools
+
+In order to spin up a full stack of a working API and other [dependant
+services](./docker-compose.yml) we recommend using the [AP
```

Comment:
I took the liberty of proposing this as our position without discussing it so let's take this opportunity to discuss and disagree/agree with it?